deploy/kubernetes/base/controller/controller.yaml

```diff
@@ -21,6 +21,14 @@ spec:
             - "--v=5"
             - "--csi-address=/csi/csi.sock"
             - "--feature-gates=Topology=true"
+            - "--enable-leader-election=true"
+            - "--leader-election-type=leases"
+            - --leader-election-namespace=$(PDCSI_NAMESPACE)
+          env:
+          - name: PDCSI_NAMESPACE
```

Comment:
maybe `PD_DRIVER` is a better prefix? `CSI` is kind of just the interface, don't think its really the semantically important part of the name

Reply:
I think `PDCSI` is a concise way to refer to the driver where having a full descriptive name isn't necessary enough to overweight the benefits of a shorter name. Not a strong preference though - I'm also OK with changing it